NewSpring/ApollosProject
# (DEV-3529) [Prayer Module] Creates MyPrayer List

## DESCRIPTION

As a User, I want to be able to see my previous prayer requests so that I can see God's faithfulness in meeting prayers.

### What does this PR do, or why is it needed?

This PR creates the `MyPrayerCard` component that is the basic block displaying the prayers and also the `MyPrayers` component, which displays the prayers as a list.

### How do I test this PR?

Currently, the `MyPrayers` is housed on the Connect tab. Make sure you have a few prayers in Alpha-Rock and also be sure that they're public. (Should be moved to prayer Module once other components have been built)

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] Closes [DEV-3683](https://newspring.atlassian.net/secure/RapidBoard.jspa?rapidView=71&modal=detail&selectedIssue=DEV-3683) and [DEV-3684](https://newspring.atlassian.net/secure/RapidBoard.jspa?rapidView=71&modal=detail&selectedIssue=DEV-3684)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
